src/Conversion/KrnlToLLVM/KrnlCall.cpp

```diff
@@ -119,6 +119,13 @@ class KrnlCallOpLowering : public ConversionPattern {
       parameterTypeList.emplace_back(opaquePtrTy);
       parameterList.emplace_back(omTensor);
       omTensors.emplace_back(omTensor);
+    } else if (ty.isa<NoneType>()) {
+      // generate llvm null pinter for NoneType
+      auto int8Ty = IntegerType::get(context, 8);
+      auto opaquePtrTy = getPointerType(context, int8Ty);
+      parameterTypeList.emplace_back(opaquePtrTy);
+      Value nullPtr = create.llvm.null(getI8PointerType(context));
```

Comment:
You can pass `opaquePtrTy` here instead of `getI8PointerType(context)`.

Reply:
Fixed, and confirmed that it works. Thanks.